website/content/docs/debugging.mdx

```diff
@@ -147,3 +147,15 @@ Failed to initialize build 'docker': error initializing builder 'docker': plugin
 
 you should try setting your temp directory to something shorter. This can be
 done through the `TMPDIR` environment variable.
+
+## Issues when temp directory does not allow execution
+
+Sometimes, Packer runs into trouble (eg: when installing a plugin ) because the
+system's temporary directory (/tmp) is restricted and doesn't allow programs to run there.
```

Comment:
That sentence seems a bit verbose, I would think we can make it terser without losing meaning.